tests/index.ts

```diff
@@ -538,13 +538,15 @@ export const linterCheckTest = ({
   skipTestIf = (_version?: string) => false,
   preCheck,
   postCheck,
+  versionGreaterThanOrEqual,
```

Comment:
Can you either add a comment/doc for this above (line 533), or use `customLinterCheckTest` instead (with `args: -a`)? Trying to generally keep `linterCheckTest` as simple as possible